src/engine/strat_engine/blockdevmgr.rs

```diff
@@ -66,14 +66,17 @@ pub fn map_to_dm(bsegs: &[BlkDevSegment]) -> Vec<Segment> {
 
 #[derive(Debug)]
 pub struct BlockDevMgr {
-    block_devs: Vec<StratBlockDev>,
+    block_devs: HashMap<DevUuid, StratBlockDev>,
```

Comment:
This is almost certainly a bad idea. Better to stick w/ Vec until we get a purpose built structure.

Reply:
HashMap seems to be a step in the right direction. What other container would be better? `Table` is meant for things needing lookups by multiple values, but blockdevs don't have names, just uuids.

Reply:
They have device numbers and device nodes as well.

Reply:
we're not doing lookups on those at this point.